### ISSUE TITLE
怎样才能复现出论文中的结果呢

### ISSUE THREAD

**Issue by Norman-GM** [NeuBCI/JDA#1]
您好，我们在用此代码复现 MtoO cross subject的时候遇到了如下问题，
1.训练过程中association Loss(visit Loss + walker Loss)一直不改变，但是domain Loss 和 class Loss 下降，请问是什么原因呢？
2.在训练集的过程中您使用了source和target的抽样数据，测试过程中采用了source和target的全部数据，这样的做法是否会导致测试集数据包含训练集数据而使得准确率增加呢？
3.在训练集中包含大量的target的数据，测试过程中也包含target的数据，这样的做法是怎样体现跨被试的？（我们理解的跨被试实验是训练中完全不包含target数据或者包含少量的target数据用于微调网络。）
我们的邮箱是1076804134@qq.com和425289167@qq.com
期待您的回复，谢谢！

Hello, we encountered the following problems when using this code to reproduce the MtoO cross subject,
1. During the training, the association Loss (visit Loss + walker Loss) has not changed, but the domain Loss and class Loss have dropped. What is the reason?
2. In the process of training set, you use the sampled data of source and target, and all the data of source and target are used in the test process. Will this approach cause the test set data to include training set data and increase the accuracy?
3. The training set contains a large amount of target data, and the test process also contains target data. How does this approach reflect cross-subjects? (The cross-subject experiment we understand is that the training does not contain target data at all or contains a small amount of target data for fine-tuning the network.)
Our mailbox is 1076804134@qq.com and 425289167@qq.com
Looking forward to your early reply, thank you!

**Comment by jiang18-318** [comment]
请问您这边复现出来了吗？

**Comment by Norman-GM** [comment]
您好，没有复现出来，后来放弃了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年9月18日(周日) 上午9:27
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [NeuBCI/JDA] 怎样才能复现出论文中的结果呢 (#1)




 
请问您这边复现出来了吗？
